### DISCUSSION TITLE
Control questions

### DISCUSSION THREAD

**Discussion by lbeltrame** [vladmandic/sdnext#2744] [Q&A] [answered]
Hello,

I'm asking here before filing enhancement issues as I don't know if some features of ControlNet for the original backend apply to the Control pipeline.

1. Does Control have the equivalent of "ControlNet is more important" / "My prompt is more important" / "Balanced" found in the ControlNet extension? If not, it is just missing, or is Control too fundamentally different?
2. A trick I use to get difficult-to-generate items in the hands of my characters is to provide (coupled with a full pose in Openpose)  a canny or a lineart representation of them (without anything else, e.g. lineart of a lone hand holding a flashlight), use the appropriate CN model, and put the CN weight above 1 (1.1 or 1.2). I noticed the Control strength goes  up to 1 only. Again, is it worth filing an enhancement issue, or is it like that by design?

Thanks!

**Comment by vladmandic** [comment, marked as answer]
re:1 - not yet, i need to investigate
re:2 - i'll increased the max strength to 2

**Comment by lbeltrame** [comment]
Thanks!
 I'm about to switch to dev to try it out (I built quite an extensive pose library for myself), I'll make sure to note things / file issues (if there are any, we all know software is perfect ;)  as I try and break things.

EDIT: I forgot, should 1. be filed as an issue to keep track "officially" or no need for it?

**Reply by vladmandic** [reply]
No harm in filing a feature request for that.
